src/sqlfluff/core/parser/segments/raw.py

```diff
@@ -120,6 +120,16 @@ def segments(self):
         """
         return []
 
+    @property
```

Comment:
Is this worth caching?

Reply:
If the things which call it are cached - probably not.